build.yaml

```diff
@@ -1,8 +1,8 @@
 nodejs:
   - "0.10"
   - "0.12"
-  - "iojs-3"
-  - "4.0"
+  - "4.3"
```

Comment:
In nvm you can just use "4" and it will install latest of the major version (4.4.1) , I don't know in nodeenv, though